Extension/lib/pages/options.js

```diff
@@ -465,6 +467,30 @@ PageController.prototype = {
         contentPage.sendMessage({type: 'clearWhiteListFilter'});
     },
 
+    _checkSafariContentBlockerRulesLimit: function (rulesCount) {
+        if (environmentOptions.isSafariBrowser) {
+            if (rulesCount > 50000) {

```

Comment:
I thought we have a constant field with this limit somewhere?
We should reuse it here.